### ISSUE TITLE
RMS bug

### ISSUE THREAD

**Issue by AdityaMH** [nikita36078/J2ME-Loader#536]
I found an midlet app when used twice or more, the screen become black screen and not responded . And the app can run normally only when clearing midlet data before start that midlet. Even though, on original j2me phone, that app run normally no matter how many i using that app.

**Comment by nikita36078** [comment]
I think it's better to fix it then 😃 What app is it?

**Comment by AdityaMH** [comment]
This app is multiscreen, maybe original screen is 240×320.
And no problem if you dont want to fix it.
[AQ30_id.zip](https://github.com/nikita36078/J2ME-Loader/files/3292953/AQ30_id.zip)

P. S.: I tested again, and become black screen when app run on third times or more (not twice).

**Comment by nikita36078** [comment]
I found what causes this bug, will fix it soon.

**Comment by nikita36078** [comment]
The issue turned out to be more serious than I expected, so I'm still trying to find a proper way to fix it. Probably that's also the reason why Tankzors is able to load save only after launching a level.

**Comment by AdityaMH** [comment]
So will be hard to be fixed?

**Comment by oatmeal01** [comment]
hello nikita, can you fix the hang problem after closing a game from menu. The phone turns into black screen then hangs, so it forces you to turn off/restart the phone.

**Comment by nikita36078** [comment]
@oatmeal01 Please create separate issue.

**Comment by nikita36078** [comment]
Fixed
